vopr.bat

```diff
@@ -0,0 +1,25 @@
+@echo off
+
+:: Install Zig 0.8.0 if a zig folder does not already exist:
+if not exist zig\ (
+    call scripts/install_zig.bat 0.8.0
+    echo Running the TigerBeetle VOPR for the first time...
+    echo Visit https://www.tigerbeetle.com
+)
+
+:: If a seed is provided as an argument then replay the seed, otherwise test 1,000 seeds:
+if not "%~1"=="" (
+    :: Build in fast ReleaseSafe mode if required, useful where you don't need debug logging:
+    if "%~2"=="-OReleaseSafe" (
+        echo Replaying seed %~1 in ReleaseSafe mode...
+        call zig run src/simulator.zig -OReleaseSafe -- %~1
+    ) else (
+        echo Replaying seed %~1 in Debug mode with full debug logging enabled...
+        call zig run src/simulator.zig -ODebug -- %~1
+    )
+) else (
+    call zig build-exe src/simulator.zig -OReleaseSafe

```

Comment:
Change to; `call zig\zig build-exe src\simulator.zig -OReleaseSafe`
We need to point to the `zig` instance that was just installed `(0.8.0)`

### Otherwise; 
![2Capture](https://user-images.githubusercontent.com/1992108/134458934-7f653463-efeb-4a14-aa17-a17b2ea29e04.PNG)
.